test/mibs.jl

```diff
@@ -0,0 +1,342 @@
+module TestJuMPInput
+
+using BilevelJuMP
+using JuMP
+using Test
+
+function runtests()
+    for name in names(@__MODULE__, all = true)
+        if startswith("$(name)", "test_")
+            @testset "$(name)" begin
+                getfield(@__MODULE__, name)()
+            end
+        end
+    end
+end
+
+function test_basic_example_1()
+    model = BilevelModel()
+    @variable(Upper(model), y, Int)
+    @variable(Upper(model), z, Int)
+    @variable(Lower(model), x, Int)
```

Comment:
TODO 1:  add an error message if not all variables are binary or integer to avoid MibS infinite loop.